glyph-brush-layout/src/characters.rs

```diff
@@ -12,6 +12,7 @@ use std::{
 };
 
 /// Single character info
+#[derive(Clone, Debug)]
```

Comment:
I don't think we should add derived implementations to private types that aren't needed. So can you remove all the private derives you've added?